### ISSUE TITLE
Portable on Windows 10 is not working

### ISSUE THREAD

**Issue by mike2003** [giuspen/cherrytree#1929]
**Version, Operative system**
Сherrytree_0.99.44.0 Windows 10 x64

**Describe the bug**
The program does not save the settings in its folder.

I tried this
**"for portable config put config.cfg in portable folder root, beside license.txt"**
styles\
mingw64\
etc\
config.cfg
license.txt
But when I run the program mingw64/bin/cherrytree.exe, it looks for settings here:
C:\Users\***\AppData\Local\cherrytree\config.cfg

**Comment by giuspen** [comment]
Are you sure that you are not hiding the extension and the file is in fact config.cfg.txt?

**Comment by mike2003** [comment]
as you can see above - no

**Comment by dardoor** [comment]
I can't reproduce with the latest version. Make sure the CherryTree portable folder is not 'read only' - this can happen sometimes when copying files between systems.

**Comment by mike2003** [comment]
No, everything is as it should be. I renamed the folder and everything stopped working.
